core/unit_test/TestMDRangePolicyConstructors.hpp

```diff
@@ -138,4 +138,43 @@ TEST(TEST_CATEGORY_DEATH, policy_invalid_bounds) {
 }
 #endif
 
+TEST(TEST_CATEGORY, policy_get_tile_size) {
+  constexpr int rank = 3;
+  using Policy    = Kokkos::MDRangePolicy<TEST_EXECSPACE, Kokkos::Rank<rank>>;
+  using tile_type = typename Policy::tile_type;
+
+  std::size_t last_rank =
+      (Policy::inner_direction == Kokkos::Iterate::Right) ? rank - 1 : 0;
+
+  auto default_size_properties =
+      Kokkos::Impl::get_tile_size_properties(TEST_EXECSPACE());
```

Comment:
I am confused about what that test is trying to achieve.

Reply:
The main purpose of this test is to verify that the new functions are correctly returning the default tile sizes that are internally used during the construction of `MDRangePolicy`. Some of these are used from the values set in a function for each backend:

https://github.com/kokkos/kokkos/blob/98b1a38e5391c57b866c4b16bf8108fce56d827a/core/src/KokkosExp_MDRangePolicy.hpp#L143-L152

with exception to the last rank of the policy.
So, the test is checking against the internal value to confirm that the function will be returning the right values.